examples/pytorch/cpu-demo/demo.yaml

```diff
@@ -13,7 +13,7 @@ spec:
           containers:
             - name: pytorch
               image: pytorch-cpu:py3.8
-              imagePullPolicy: Always
+              imagePullPolicy: IfNotPresent
```

Comment:
You can remove this imagePullPolicy, since IfNotPresent is set by default in Kubernetes container.